server/services/service.js

```diff
@@ -81,7 +81,9 @@ module.exports = () => ({
       startDate: x[config.startField],
       endDate: config.endField
         ? x[config.endField]
-        : moment(x[config.startField]).add(config.defaultDuration, 'minutes'),
+        : config.defaultDuration === 'fullDay' 
```

Comment:
default duration will never be fullDay, you would need to compare to that value which is `1440`